.github/workflows/lint.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-go@v4
         with:
-          go-version: "1.20"
+          go-version: "1.21"
```

Comment:
can you also bump golangci-lint to  version: "v1.55"?
thanks

Reply:
Done